templates/src/app/fileupload/index.js

```diff
@@ -0,0 +1,43 @@
+var { Router } = require("express");
```

Comment:
Mind renaming this from `fileupload` to `file`? It handles both upload and download.